cv-data-service-library/src/main/java/com/trihydro/library/helpers/CdotGisConnector.java

```diff
@@ -0,0 +1,74 @@
+package com.trihydro.library.helpers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import com.trihydro.library.service.RestTemplateProvider;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CdotGisConnector {
+  private final String baseUrl = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes/MapServer/exts/CdotLrsAccessRounded";
```

Comment:
chore: move this to a configuration value so that, in the off chance this value changes, we don't have to recompile the whole codebase. This is somewhat of a magic string right now https://en.wikipedia.org/wiki/Magic_string

Reply:
As we discussed offline, we will keep the base URL as a hardcoded string.